data/sql/updates/pending_db_world/rev_1655187726583284000.sql

```diff
@@ -0,0 +1,185 @@
+(45811, 3236, 0, 0, 1, 0, 0, 1, 1, 0, -3798.8208, -1829.7017, 94.17238, 4.366050243377685546, 600, 0, 0, 1, 0, 0, 0, 0, 0, '', 0),
+(45812, 3236, 0, 0, 1, 0, 0, 1, 1, 0, -3807.5981, -1865.3707, 92.56358, 1.319851279258728027, 600, 0, 0, 1, 0, 0, 0, 0, 0, '', 0),
+/* Combo 2 */
+(45813, 3235, 3236, 0, 1, 0, 0, 1, 1, 0, -3787.7153, -1848.3755, 93.918015, 2.55345463752746582, 900, 0, 0, 1, 0, 2, 0, 0, 0, '', 0), -- .go xyz -3787.7153 -1848.3755 93.918015 Leader 
+(45814, 3237, 3236, 0, 1, 0, 0, 1, 1, 0, -3814.7678, -1845.9219, 93.61283, 0.949117600917816162, 900, 0, 0, 1, 0, 0, 0, 0, 0, '', 0),
+(45815, 3237, 3236, 0, 1, 0, 0, 1, 1, 0, -3798.8208, -1829.7017, 94.17238, 4.366050243377685546, 900, 0, 0, 1, 0, 0, 0, 0, 0, '', 0),
+(45816, 3235, 3236, 0, 1, 0, 0, 1, 1, 0, -3807.5981, -1865.3707, 92.56358, 1.319851279258728027, 900, 0, 0, 1, 0, 0, 0, 0, 0, '', 0),
+/* Combo 3 */
+(45817, 3237, 0, 0, 1, 0, 0, 1, 1, 0, -3787.7153, -1848.3755, 93.918015, 2.55345463752746582, 1800, 0, 0, 1, 0, 2, 0, 0, 0, '', 0), -- .go xyz -3787.7153 -1848.3755 93.918015 Leader 
+(45818, 5827, 0, 0, 1, 0, 0, 1, 1, 0, -3814.7678, -1845.9219, 93.61283, 0.949117600917816162, 1800, 0, 0, 1, 0, 0, 0, 0, 0, '', 0),
+(45819, 3235, 0, 0, 1, 0, 0, 1, 1, 0, -3798.8208, -1829.7017, 94.17238, 4.366050243377685546, 1800, 0, 0, 1, 0, 0, 0, 0, 0, '', 0),
+(45820, 3236, 0, 0, 1, 0, 0, 1, 1, 0, -3807.5981, -1865.3707, 92.56358, 1.319851279258728027, 1800, 0, 0, 1, 0, 0, 0, 0, 0, '', 0);
+
+DELETE FROM `pool_creature` WHERE `guid` IN (45809, 45810, 45811, 45812, 45813, 45814, 45815, 45816, 45817, 45818, 45819, 45820);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `description`) VALUES 
+(45809, 391, 'Brontus Kodo Pack Combo 1 -- 4 Barrens Kodo Pack Leader Mob'),
+(45810, 391, 'Brontus Kodo Pack Combo 1 -- 4 Barrens Kodo Pack'),
+(45811, 391, 'Brontus Kodo Pack Combo 1 -- 4 Barrens Kodo Pack'),
+(45812, 391, 'Brontus Kodo Pack Combo 1 -- 4 Barrens Kodo Pack'),
+(45813, 392, 'Brontus Kodo Pack Combo 2 -- Mixed Kodo Pack Leader Mob'),
+(45814, 392, 'Brontus Kodo Pack Combo 2 -- Mixed Kodo Pack'),
+(45815, 392, 'Brontus Kodo Pack Combo 2 -- Mixed Kodo Pack'),
+(45816, 392, 'Brontus Kodo Pack Combo 2 -- Mixed Kodo Pack'),
+(45817, 393, 'Brontus Kodo Pack Combo 3 -- Brontus Pack Leader Mob'),
+(45818, 393, 'Brontus Kodo Pack Combo 3 -- Brontus Pack'),
+(45819, 393, 'Brontus Kodo Pack Combo 3 -- Brontus Pack'),
+(45820, 393, 'Brontus Kodo Pack Combo 3 -- Brontus Pack');
+
+DELETE FROM `pool_pool` WHERE `pool_id` IN (391, 392, 393);
+INSERT INTO `pool_pool` (`pool_id`, `mother_pool`, `chance`, `description`) VALUES 
+(391, 394, 55, 'Brontus Kodo Pack Combo 1 -- 4 Barrens Kodo Pack'),
+(392, 394, 30, 'Brontus Kodo Pack Combo 2 -- Mixed Kodo Pack'),
+(393, 394, 15, 'Brontus Kodo Pack Combo 3 -- Brontus Pack');
+
+-- Pathing for Kodos with Brontus
+SET @NPC := 45809;
+SET @path := @NPC * 10;
```

Comment:
Path should be on upper case, but you can leave it as it is for this one if you want.